Libraries/Components/WebView/WebView.ios.js

```diff
@@ -98,6 +98,22 @@ var WebView = React.createClass({
      */
     renderLoading: PropTypes.func,
     /**
+     * Invoked when load finish
+     */
```

Comment:
Oh yeah, it's github being annoying by still showing outdated comments. I was thinking of deleting all the previous comments when receiving a new update to improve the situation